src/models/CronTabModel.ts

```diff
@@ -0,0 +1,25 @@
+import { modelToRef } from '@crontab-utils/utils';
+import { K8sResourceCommon } from '@openshift-console/dynamic-plugin-sdk';
+import { K8sModel } from '@openshift-console/dynamic-plugin-sdk/lib/api/common-types';
+
+export const CronTabModel: K8sModel = {
+  label: 'CronTab',
+  labelPlural: 'CronTabs',
+  apiVersion: 'v1',
+  apiGroup: 'stable.example.com',
+  plural: 'crontabs',
+  abbr: 'ct',
+  namespaced: true,
+  kind: 'CronTab',
+  id: 'crontab',
+  crd: true,
+};
+
+export type CronTabKind = K8sResourceCommon & {
+  spec?: {
+    [key: string]: any;
+  };
+  status?: { [key: string]: any };
+};
+
+export const cronTabModelRef = modelToRef(CronTabModel);
```

Comment:
We intended to remove the concept of resource references from the SDK... Where is this needed?

Reply:
this was used when manually building the URLs

Reply:
So Im not sure how to avoid building the url manually at this point.
I start to think this is a tech debt on our side and we should expose a function which will help plugins creators with crafting URLs.
Thoughts?

Reply:
Yeah, I think we need an SDK method for building resource URLs. Then if we ever need to change how the URLs are built (e.g. if we remove the API resource version from the URL) plugins aren't broken.